.github/workflows/clubhouse.yml

```diff
@@ -14,3 +14,10 @@ jobs:
           opened-state-name: In Development
           merged-state-name: Completed
           ignored-users: geminihlswadmin
+          user-map: |
```

Comment:
cquiroz is the default?

Reply:
That's the format suggested in the action documentation
https://github.com/singingwolfboy/create-linked-clubhouse-story#user-map

Reply:
We are using a token created by me. I think that's the reason all tickets endup under my name